package.json

```diff
@@ -11,9 +11,12 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "axios": "^0.21.1",
     "debug": "^4.1.1",
     "js-conflux-sdk": "^1.5.11",
+    "json-rpc-engine": "^6.1.0",
     "lodash": "^4.17.21",
+    "pify": "^5.0.0",
```

Comment:
Seems axios and pify is not used in this project right ?

Reply:
rm unused modules